app/controllers/users_controller.rb

```diff
@@ -23,10 +33,53 @@ def create
     end
   end
 
+  def edit; end
+
+  def update
+    if @user.update(user_params)
```

Comment:
```suggestion
    if @user.update user_params
```
so sánh update, update_attributes, update_attribute, update_column, update_columns

Reply:
-update:(name: value)
Là một instance method
Cập nhật một hoặc nhiều thuộc tính của bản ghi
Kiểm tra validations, chạy callback khi cập nhật
Cập nhật thành công trả về True, ngược lại trả về False
Trường updated_at được cập nhật nếu cập nhật thành công
update! raise exception nếu cập nhật thất bại

-update_attribute
Chỉ cập nhật một thuộc tính của bản ghi
Bỏ qua validations, vẫn chạy callback
Trả về True/False
Trường updated_at được cập nhật

-update_attributes
Giống với update(name: value)
update_attributes! raise exception nếu cập nhật không thành công

-update_column
Cập nhật một thuộc tính trực tiếp trong cơ sở dữ liệu
Validations, callback đều bị bỏ qua khi cập nhật
Trả về True/False
Trường updated_at không được cập nhật

-update_columns
Cập nhật một hoặc nhiều thuộc tính trực tiếp trong cơ sở dữ liệu
Bỏ qua validations, không chạy callback
Trả về True/False
Trường updated_at không được cập nhật